workspace-hack/Cargo.toml

```diff
@@ -17,6 +17,8 @@ publish = false
 bitflags = { version = "2", default-features = false, features = ["serde", "std"] }
 bytes = { version = "1", features = ["serde"] }
 chrono = { version = "0.4", features = ["serde"] }
+clap = { version = "4", features = ["cargo", "derive", "env", "wrap_help"] }
+clap_builder = { version = "4", default-features = false, features = ["cargo", "color", "env", "std", "suggestions", "usage", "wrap_help"] }
```

Comment:
What are you sneaking in here with these changes?

Reply:
Nothing, I hope! I needed to update the `clap` dependency along with the other changes, to satisfy Cargo. Hope that's not a problem?